umei-openapi.json

```diff
@@ -1636,7 +1648,72 @@
           }
         },
         "additionalProperties": false
-      }
+      },
+      "ProblemDetails": {
+        "type": "object",
+        "description": "An object describing the problem in a machine-and-user readable version, based on / extending RFC7807. ",
+        "additionalProperties": true,
+        "properties": {
+          "type": {
+            "type": "string",
+            "description": "A URI reference [RFC3986] that identifies the\n      problem type.  This specification encourages that, when\n      de-referenced, it provide human-readable documentation for the\n      problem type (e.g., using HTML [W3C.REC-html5-20141028]).  When\n      this member is not present, its value is assumed to be\n      \"about:blank\"",
+            "nullable": false,
+            "example": "https://umei.org/error-codes/invalid-grid-reference"
+          },
+          "title": {
+            "type": "string",
+            "description": "A short, human-readable summary of the problem\n      type.  It SHOULD NOT change from occurrence to occurrence of the\n      problem, except for purposes of localization (e.g., using\n      proactive content negotiation;",
+            "example": "The grid node id is not found or not valid for this operation"
+          },
+          "status": {
+            "type": "string",
+            "description": "The HTTP status code generated by the origin server for this occurrence of the problem. ",
+            "example": "400",
+            "nullable": true
+          },
+          "detail": {
+            "type": "string",
+            "description": "A human-readable explanation specific to this\n      occurrence of the problem.",
```

Comment:
I don't know where the "\n" are coming from but I don't see any impact when visualising it in swagger editor

Reply:
I removed then anyway